PWGHF/Tasks/HFMCValidation.cxx

```diff
@@ -174,6 +178,14 @@ struct ValidationGenLevel {
           }
           double pDiff = RecoDecay::p(pxDiff, pyDiff, pzDiff);
           double ptDiff = RecoDecay::pt(pxDiff, pyDiff);
+          auto daughter0 = particle.daughters_as<aod::McParticles>().begin();
+          double vertexDau[3] = {daughter0.vx(), daughter0.vy(), daughter0.vz()};
+          double vertexPrimary[3] = {mccollision.posX(), mccollision.posY(), mccollision.posZ()};
+
+          auto decayLength = RecoDecay::distance(vertexPrimary, vertexDau);
+          if (whichHadron == 1 && origin == OriginType::Prompt && decayLength * 10000 > 100) {
+            RecoDecay::getMother(particlesMC, particle, kBottom, true, nullptr, -1, true);
```

Comment:
The last `true` does not match any argument of `getMother`.
Also you don't do anything with the returned index.

Reply:
You are right, I added a feature for debugging and I forgot to remove it, now fixed 👍